tools/package-tools/build-release.ts

```diff
@@ -17,7 +20,7 @@ const bundlesDir = join(outputDir, 'bundles');
  * release folder structure. The output will also contain a README and the according package.json
  * file. Additionally the package will be Closure Compiler and AOT compatible.
  */
-export function composeRelease(packageName: string) {
+export function composeRelease(packageName: string, options: ReleaseBuildOptions = {}) {
```

Comment:
Maybe `ReleaseComposeOptions` here? To be consistent with the function name. Or we change `composeRelease` to `buildRelease`.